layers/+lang/reason/README.org

```diff
@@ -0,0 +1,70 @@
+#+TITLE: Reason layer
+
+[[https://reasonml.github.io/img/reason.svg]]
+
+* Description
+
+Spacemacs layer for ReasonML, based around [[https://github.com/reasonml-editor/reason-mode][reason-mode]].
+
+** Features:
+  - Syntax highlighting / indentation
+  - Autocomplete (via `merlin`)
+  - Lint / error display (via `merlin`, and `flycheck` if `syntax-checking` layer is enabled)
+  - REPL via `rtop`
+  - Auto-formatting (via `refmt`)
+
+* Install
+
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =reason= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+This layer depends on the `ocaml` layer which should be installed automatically.
+
+** Merlin
+
+You'll need merlin (binaries `ocamlmerlin` and `ocamlmerlin-reason`) installed
+on your system and on your `PATH` for errors and autocompletion.
+
+If you're familiar with `opam` you can `opam install merlin reason` to get
+`ocamlmerlin` and `ocamlmerlin-reason` on your current `opam` switch.
+
+** Refmt
+
+Similary to Merlin, you'll need the `refmt` binary on your path.
+
+If you're familiar with `opam` you can `opam install reason` to get `refmt` on
+your current `opam switch`.
+
+If your project depends on a specific version of `refmt`, you can set
+`refmt-command` (via `customize-mode` or `(setq refmt-command ..)`) to the path
+to a particular binary. You can also use the special values `'npm` or `'opam` to
+run `refmt` via `npx` or `opam exec`, which will keep you on the correct version
```

Comment:
typo: you -> your
#### before:
for you current opam switch

#### after:
for your current opam switch